File: a2m-afp-front/src/model/pro/ProjectDto.ts
export class ProjectDto {
    id: number | null
    title: string | null
    description: string | null
    projectType: string | null
    uri: string | null

    constructor(id: number | null, projectType: string | null, title: string | null, uri: string | null, description: string | null) {
        this.id = id
        this.projectType = projectType
        this.title = title
        this.uri = uri
        this.description = description
    }
}

File: a2m-afp-back/src/main/java/egovframework/com/a2m/egov/model/afp/project/ProjectDto.java
/**
 *
 */
package egovframework.com.a2m.egov.model.afp.project;

import java.io.Serializable;
import java.util.Date;
import javax.validation.constraints.NotEmpty;
import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @author tiennd
 *
 * @created Apr 18, 2023
 */

@Getter
@Setter
@NoArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
public class ProjectDto implements Serializable {
	private static final long serialVersionUID = 253L;
	private Long id;
	@NotEmpty
	private String title;
	@NotEmpty
	private String description;
	@NotEmpty
	private String projectType;
	private String uri;
	private Long viewNumber;
	private String isBookmark;
	private String crud;
	private String postType;
	private String createdBy;
	private String createdByNm;
	private String updatedBy;
	private String updatedByNm;
	private String imageUrl;
	private Date createdDate;
	private Date updatedDate;
}
